packages/server/graphql/mutations/helpers/notifications/MattermostNotifier.ts

```diff
@@ -387,5 +392,24 @@ export const MattermostNotifier: Notifier = {
 
   async integrationUpdated(dataLoader: DataLoaderWorker, teamId: string, userId: string) {
     ;(await getMattermost(dataLoader, teamId, userId))?.integrationUpdated()
+  },
+
+  async standupResponseSubmitted(
+    dataLoader: DataLoaderWorker,
+    meetingId: string,
+    teamId: string,
+    userId: string
+  ) {
+    const {meeting, team} = await loadMeetingTeam(dataLoader, meetingId, teamId)
```

Comment:
same parallel pattern for this one

Reply:
Also fixed (and in the slack notifier, too)